File: 프로그래머스/0/181838. 날짜 비교하기/날짜 비교하기.js
function solution(date1, date2) {
    var answer = 0;
    if (date2[0] > date1[0]) {
        answer = 1
    }
    else if (date2[0] === date1[0] && date2[1] > date1[1]){
        answer = 1
    }
    else if (date2[0] === date1[0] && date2[1] === date1[1] && date2[2] > date1[2]){
        answer = 1 
    }
    return answer;
}

File: 프로그래머스/0/181838. 날짜 비교하기/날짜 비교하기.py
def solution(date1, date2):
    answer = 0
    if date2[0] > date1[0]:
        answer = 1
    if date2[0] == date1[0] and date2[1] > date1[1]:
        answer = 1
    if date2[0] == date1[0] and date2[1] == date1[1] and date2[2] > date1[2]:
        answer = 1
    return answer